templates/_helpers.tpl

```diff
@@ -61,3 +61,29 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Traverse all binary assets and create path names for them with data
+*/}}
+{{- define "superset.nestedAssets" -}}
```

Comment:
This was kind of fun, but a total abuse of the template system - it's pretty crazy though that there's no? way to iterate nested keys of a map.  Also, go templates don't support "break" or "continue" for silly reasons.